db/migrate/20200423070209_create_notifications.rb

```diff
@@ -0,0 +1,8 @@
+class CreateNotifications < ActiveRecord::Migration[6.0]
+  def change
+    create_table :notifications do |t|
+
```

Comment:
What? :D
In this case should not create the table

Reply:
i dont now which fields i have to create here=\
and how it resource will looks

Reply:
Then you shouldn't create the table